.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
I totally agree with you, in keeping the logic in one place.
I wish the tito scripts were in bash so I could write a wrapper function.  What I have below isn't pretty, but it get's the job done.